File: 프로그래머스/0/181891. 순서 바꾸기/순서 바꾸기.py
def solution(num_list, n):
    # n번째 이후의 요소 구하기
    slice_list = num_list[:n]
    
    # n번째 까지의 원소 구하기 
    slice_list2 = num_list[n:]
    
    # 두 리스트 이어붙여서 반환
    return slice_list2 + slice_list

File: 프로그래머스/0/181891. 순서 바꾸기/순서 바꾸기.java
import java.util.Arrays;

public class Solution {
    public static int[] solution(int[] num_list, int n) {
        int len = num_list.length;
        int[] answer = new int[len];

        // 뒤쪽 부분 복사 (n번째 이후)
        int index = 0;
        for (int i = n; i < len; i++) {
            answer[index++] = num_list[i];
        }

        // 앞쪽 부분 복사 (n번째까지)
        for (int i = 0; i < n; i++) {
            answer[index++] = num_list[i];
        }

        return answer;
    }
}
